scss/boost_union/post.scss

```diff
@@ -917,3 +914,994 @@ body.pagelayout-login:not(.loginbackgroundimage) #footnote {
         }
     }
 }
+
+/* ========================
+ Smartmenu: style.
+ ========================== */
+
+/* Scrollbar */
+
+/* Width and height of the scrollbar */
+::-webkit-scrollbar {
+    width: 12px;
+    height: 10px;
+}
+/* Background color for the scrollbar on hover */
+::-webkit-scrollbar-thumb:hover {
+    background-color: #495057;
+}
+/* Border, border radius and background color for the bar in the scrollbar */
+::-webkit-scrollbar-thumb {
+    background-color: #6a737b;
+    border-radius: 20px;
+    border: 2px solid #f8f9fa;
+}
+/* Background for the scrollbar */
+::-webkit-scrollbar-track {
+    background: #f8f9fa;
+}
+/* End of Scrollbar */
+
+/* Header Menu */
+.navbar {
+    &.fixed-top {
+        /* Added Height and background color for the top header,
+        z-index is added to show the top header dropdown menu above the header block */
+        &.boost-union-menubar { /* boost-union-menubar class is added for the top header */
+            height: 42px;
+            border-bottom: 0;
+            background-color: #f8f9fa;
+            z-index: 1031;
+            /* The position of the side drawer has been adjusted slightly below to accommodate the top header. */
+            ~ .drawer.drawer-left,
+            ~ .drawer.drawer-right {
+                top: 100px;
+            }
+           /* The position of the main page has been adjusted slightly below to accommodate the top header. */
```

Comment:
Just one space more to the right please.

Reply:
Thanks, Luca.
I fixed this in a review commit now.